server/graphql/v2/query/LatestChangelogPublishDateQuery.js

```diff
@@ -0,0 +1,31 @@
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import cache, { fetchCollectiveId } from '../../../lib/cache';
+import sequelize from '../../../lib/sequelize';
+import models from '../../../models';
+
+const LatestChangelogPublishDateQuery = {
+  type: GraphQLDateTime,
+  async resolve() {
+    const cacheKey = 'latest_changelog_publish_date';
+    let latestChangelogPublishDate = await cache.get(cacheKey);
+    if (!latestChangelogPublishDate) {
+      const collectiveId = await fetchCollectiveId('opencollective');
+      latestChangelogPublishDate = await models.Update.findOne({
```

Comment:
The variable should not be named `latestChangelogPublishDate` if it doesn't contain a date.

Reply:
You see the problem when you read this `latestChangelogPublishDate?.date`. Why read the date of a date?

Reply:
Thanks. Corrected it. 🐨